docs/validators/Missed_Blocks.md

```diff
@@ -0,0 +1,36 @@
+# Missed Blocks
+
+- [Missed Blocks](#missed-blocks)
+  * [DB Changes](#db-changes)
+  * [Ingestion](#ingestion)
+  * [Query Updates](#query-updates)
+
+Currently we are pulling the `missed blocks` count from signing infos. Unfortunately, it is not a running count. This 
```

Comment:
This count is still useful for showing how close a validator is to being slashed (you are allowed to have this count up to 50% of our missed-block-count config before being penalized).  Perhaps a separate future item might leverage this to show how close an offline validator (or partially down validator) is to being thrown in jail

Reply:
If you wanna add an issue to explain a little more, I can add that to the next release